Mobile pages: Gallery, Settings

sprice opened the pull request:
Most of the work here is making the Gallery mobile friendly.

On small screens we are only showing the carousel (we hide the tiles below).

The Settings page has also been updated, though it will break on the smallest of screens if the user has the ability to add an item to the Gallery.

A few markup changes were also made to the Manage Tokens page to make it more mobile friendly. We won't put in more work here as this page will be revamped soon.

Anaphase commented on src/views/GalleryView.vue:
```diff
@@ -102,18 +108,19 @@ export default {
 // a lot of these .carousel-container styles are mainly here for the fullscreen
 //  view
 .carousel-container
-  width: 100%
```
This is needed for the fullscreen view to work right

Anaphase commented on src/views/GalleryView.vue:
```diff
@@ -102,18 +108,19 @@ export default {
   background-color: $color-dark
 
 .carousel
-  width: 50%
```
For the `.fixed-size-carousel` stuff to work right, this container MUST have a defined width & height. Removing this on mobile makes differently sized image have huge, jarring content jumps between slides (which is the whole reason I made the `.fixed-size-carousel` stuff)

sprice replied:
Width has been added back.

Anaphase replied:
A width _and_ height must be defined. The height is especially important because it's what prevents the content jumps.

sprice commented:
@Anaphase:

1) I updated the bullets on `FaucetMarketingCard` so that more text would fit horizontally, and also as bullets should be hung. It does look like the bullets themselves could be brought a little closer to the text, and further away from the left edge

2) I updated the giveaway card to fix the button on small widths.

3) I added `align-items: start` and things look a lot better.

sprice commented on src/components/GalleryListItem.vue:
```diff
@@ -50,18 +51,30 @@ export default {
-.gallery-card
-  width: 25%
-  margin-bottom: 2em
+.card
```
done.

Anaphase commented:
@sprice it needs a width _and_ a height, though. Also, the `.carousel-container` on the Gallery page needs to be 100% width & height for the fullscreen stuff to look right. I have a bunch of pending changes right now so I can't switch my branch, but as long as it looks right in fullscreen and there are no content jumps between differently sized images then I guess it's okay to omit heights (but I kinda doubt it does.)

Anaphase commented on src/views/ContactUsView.vue:
```diff
@@ -0,0 +1,79 @@
+  data() {
+    return {
+      supportHref: `mailto:support@codexprotocol.com?subject=${encodeURIComponent('Codex Viewer Support')}`,
+      developersHref: `mailto:support@developers.com?subject=${encodeURIComponent('Codex Viewer Developer Support')}`,
```
```suggestion
      developersHref: `mailto:developers@codexprotocol.com?subject=${encodeURIComponent('Codex Viewer Developer Support')}`,
```

johnhforrest replied:
Too much copy paste!
